### ISSUE TITLE
Doesn't Work In Search Results

### ISSUE THREAD

**Issue by xnamkcor** [edcaron/Reddit-Link-Opener-Firefox#1]
Make it work on lists of Reddit posts when displayed in a Reddit search.

**Comment by edcaron** [comment]
The search results are on a different layout than the posts on the subreddit, it would require a complete re-write of the code for it to work on both. 
I don't have the time do to it right now but feel free to send a pull request and I'll look into it!

**Comment by xnamkcor** [comment]
I assumed the script worked by just looking for all instances of the page that looked like a post link or a comment link. Sorry. At least you have the Comments Only feature I asked for a while ago. Thanks for that.

**Comment by edcaron** [comment]
Yeah the comments only it's a feature I also baldly wanted. The owner of the other repo didn't have time to look at the pull requests so I forked

**Comment by edcaron** [comment]
hey @xnamkcor, with the new layout fix I made the extension is working on the search page. Please, update and Try it out https://new.reddit.com/search?q=funny if you'd like.

**Comment by xnamkcor** [comment]
Can't get it to work. I tried both old and new layout(I use old by default).
